tests/e2e/features/artifacts/transferPage/atrifacts-transfers.feature

```diff
@@ -202,4 +202,30 @@ Feature: Artifacts - UI
     Then Modal card element with the "//*[contains(@class, '-my-0.5')]//button[contains(., 'ETH')]" xpath should be "visible"
     Then Element with "text" "Send to zkSync Era Testnet" should be "visible"
     Then Element with "text" "Send to zkSync Era Testnet" should be "clickable"
-    
\ No newline at end of file
+
+  @id1286 @Transfer
+  Scenario: Check max button functionality
+    When I go to page "/transaction/zksync/era/send/?address=0x2CF4F78917A16C9584AeB5d4c5bD2713d724C75d"
+    Then I confirm the network switching
+      #Max button is displayed
+    Then Element with "partial text" "Max" should be "visible"
+    Then Element with "partial text" "Max" should be "clickable"
+      #Verify "Max" button is highlighted after clicking on it
+    Then I click by "text" with " Max " value
+    Then Element with "partial class" "is-max" should be "visible"
+      # Verify max amount set to the field for Withdraw
+    Then Element with "title" "Max amount is set" should be "visible"
+      #Continue button is clickable
+    Then Element with "text" " Continue " should be "clickable"
+
+  @id1538 @Transfer
+  Scenario: Check Transaction submitted window redirection links
+    When I go to page "/transaction/zksync/era/send/?address=0x2CF4F78917A16C9584AeB5d4c5bD2713d724C75d"
+    # Fill all the required fields for tx and to see Submitted window
+    Then I choose "ETH" as token and insert "0.00000001" as amount
+    When I "confirm" transaction after clicking "Send to zkSync Era Testnet" button
+    #Links check
+    Then Element with "href" "https://goerli.explorer.zksync.io/address/0x2CF4F78917A16C9584AeB5d4c5bD2713d724C75d" should be "clickable"
```

Comment:
Regarding the navigation and redirection I guess we have to open this link. 
fe:
When I click by the "//*[text()=' Track status ']" text element on the Modal card
    Then New page has "https://goerli.etherscan.io/tx/" partial address
or:
When I click by the "//*[text()=' Go to Assets page ']" text element on the Modal card
    Then I am on the Main page without a NetworkSwitcher 
    -- but this not merged yet, you can find realization in  https://github.com/matter-labs/dapp-portal/pull/125 in id1396
    The main question in your case, please check carefully - "will you be able to back again to the initial page to open a new link?"

Reply:
@pcheremu, In that case we need to use Scenario Outline - because after the first page is opened - we will be redirected to another page. So we will need to rerun the test scenario. 
Or maybe there're some other options?

Reply:
I see you did it, cool!